Library/Formula/riemann.rb

```diff
@@ -19,16 +17,16 @@ def shim_script
   end
 
   def install
-    if (etc/'riemann.config').exist?
-      (prefix/'etc').install 'etc/riemann.config' => 'riemann.config.guide'
+    if (etc/"riemann.config").exist?
+      (prefix/"etc").install "etc/riemann.config" => "riemann.config.guide"
     else
-      etc.install 'etc/riemann.config'
+      etc.install
```

Comment:
I should have been clearer: `etc.install` with the argument rather than the `.exist?` check above.